zookeeper/source/zk_nn_client.cc

```diff
@@ -1320,8 +1325,7 @@ bool ZkNnClient::add_block(const std::string &file_path,
 bool ZkNnClient::add_block_group(const std::string &fileName,
                      u_int64_t &block_group_id,
                      std::vector<std::string> &dataNodes,
-                     std::vector<u_int64_t> &storageBlockIDs,
-                     uint32_t total_num_storage_blocks) {
```

Comment:
line 1335 currently depends on this parameter. Make an appropriate change there as well. i.e. put the hard code number 9 there. (RS(6,3))